test/integration.spec.ts

```diff
@@ -9,16 +10,10 @@ describe('Distribution:', function() {
   })
 })
 
-/*
 describe('CLI command:', function() {
   it('should be able to run CLI command', function() {
-    const execP = util.promisify(exec)
-
-    async function bin() {
-      const { stdout, stderr } = await execP('./bin/rlp encode "[ 5 ]"')
+    exec('./bin/rlp encode "[ 5 ]"', (_error, stdout, _stderr) => {
```

Comment:
Not sure, I was running into linting errors like ``'error' is declared but its value is never read`` here, however I didn't want to explicitly handle error and stuff here, so I googled towards the underscore solution. Is this a good way to do it?

Reply:
Yup, I do this like this myself. Furthermore: `_stderr` could be deleted entirely since it's the last parameter.